Remove context handler mocking in PDP integration tests and fixed small issues so that tests run

acn-dgopa opened the pull request:
Reset the attributes with "-" 
Added mocking for policyinformationrepository in integration tests
Removed mocking for context handler
Fixed prp tests
Added required test data

acn-sbuad commented on src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Altinn.Platform.Authorization.IntegrationTest.csproj:
```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Altinn.Authorization.ABAC\Altinn.Authorization.ABAC.csproj" />
```
I'm thinking this should be included as a nuget rather than a project reference as this is what is done for the component that's under testing

acn-sbuad replied:
Removing the line as I see the nuget is included further up

acn-dgopa replied:
this is already done by @Kritsi but my branch was created before her fix, so merging master into branch would fix it

acn-sbuad commented:
@acn-dgopa  Are you looking at the failing test or should we be fixing that ?

acn-dgopa commented:
> @acn-dgopa Are you looking at the failing test or should we be fixing that ?

I'm not looking into it @acn-sbuad , I had updated the issue with my findings on Friday. I updated @TheTechArch aswell about the issue
